cuenca/resources/card_transactions.py

```diff
@@ -0,0 +1,35 @@
+import datetime as dt
+from dataclasses import dataclass
+from typing import ClassVar, List, cast
+
+from cuenca_validations.types import (
+    CardErrorType,
+    CardNetwork,
+    CardTransactionType,
+    CardType,
+)
+
+from .base import Transaction
+from .resources import retrieve_uri
+
+
+@dataclass
+class CardTransaction(Transaction):
+    _resource: ClassVar = 'card_transactions'
+
+    type: CardTransactionType
+    network: CardNetwork
+    related_card_transaction_uris: List[str]
+    card_last4: str
+    card_type: CardType
+    merchant: str
+    metadata: dict
+    error_type: CardErrorType
+    expired_at: dt.datetime
+
+    @property  # type: ignore
+    def related_card_transactions(self) -> List['CardTransaction']:
+        return [
+            cast('CardTransaction', retrieve_uri(uri))
```

Comment:
Do this in parallel. It's a straightforward optimization for something that's going to be used frequently